modules/ROOT/pages/hdfs/hdfs-connector-studio.adoc

```diff
@@ -0,0 +1,121 @@
+
+. Create a Mule project.
+. Supply your authentication credentials to Anypoint Platform.
+. Add the connector to your Mule project
+. Configure an input source for the connector's flow.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+* Configure an associated global element for the input source and connector.
+* Create a YAML or properties file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+== Add the Connector to Your Project
+
+Add Hadoop (HDFS) Connector to a Mule project to automatically populate the XML code with the connector's namespace and schema location, as well as add the required dependencies to the project's `pom.xml` file.
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type HDFS in the search field.
+. Click the Hadoop (HDFS) connector in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+When you design your application in Studio, the act of dragging the connector's operation from the palette onto the Anypoint Studio canvas automatically populates the XML code with the connector namespace and schema location.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. Possible input source operations for this connector are:
+
+* Read +
+Start your app when the content of a file is read and streamed to the rest of the flow.
+* HTTP Listener +
+Start your app using a browser or a command such as `curl`.
+* Scheduler +
+Start your app at timed intervals.
+
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas.
+
+The required fields for the `Read` input source are:
+
+* Configuration +
+The name of the connector configuration to use
+* Path +
+The path for the file to read 
+
+[[configure_global_element]]
+== Configure a Global Element for the Connector
+
+To configure a connector in your Mule app, first configure a global element that can be used by all instances of that connector in the flow:
+
+. In the Mule Palette view, search for the connector and select an operation.
+. Drag the operation onto the canvas to the right of the input source. 
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to add a global element for the connector.
+. In the *General* tab, enter a name for the configuration, and select the connection type from *Connection*. +
+Supported connection types for this connector are: 
+* *Kerberos*
+* *Simple*
++
+For information about how to configure the connections, see <<configure_authentication,Configure Authentication>>.
```

Comment:
Hmm...I see what you did here and below.
This step should really be...:
```suggestion
. In the *General* tab, enter a name for the configuration, select either the Kerberos or Simple connection type from *Connection,* and complete the dialog box provided to configure that connection type.+
```
What would logically follow here are the substeps/values for configuring each type...but that would hugely interrupt this task. 
But as part of this task, shouldn't we present configure Kerberos and configure Simple as subtasks, instead of creating the container of Configure Authorization to hold them?